windows/installer.nsi

```diff
@@ -212,6 +218,30 @@ Section "Install_64Bit" INST_64
                 MessageBox MB_OKCANCEL|MB_ICONEXCLAMATION "$(OLD_VER_REMOVE_FAILED)" /sd IDCANCEL IDOK continueinstall
                 goto quit
             ${EndIf}
+
+            ; ask to move old ASIO4ALL registry configuration
+
+            ; check if old ASIO4ALL keys exit
+
+            ClearErrors
+            EnumRegKey $0 HKCU "SOFTWARE\ASIO4ALL v2 by Wuschel\8A9E7A56" 0
+
+            IfErrors 0 continueinstall
+                MessageBox MB_YESNO|MB_ICONEXCLAMATION "Do you want to copy ASIO4ALL settings from the previous version if possible?" /sd IDYES IDNO continueinstall
```

Comment:
If we find this key, we'll ask if he wants to copy the settings (this message doesn't come up yet.)